script/Ciel/Day 10/10_01_CIEL10_4_2C.txt

```diff
@@ -110,7 +110,7 @@ My main role as an Executor is to perform inquisitions,
  rescue people in distress,
  coordinate with other Executors,
  and perform all sorts of other duties.
- In short, I'm a jack-of-all-trades."
+ In short, I'm a jack of all trades."
```

Comment:
MW says dashes
https://www.merriam-webster.com/dictionary/jack-of-all-trades

Reply:
Hmm, ngram suggests otherwise. For future reference in cases like this, which do I follow, MW or ngram?
![image](https://user-images.githubusercontent.com/90473809/234774839-95a72b19-83df-497e-acb8-eae31319dd0b.png)

Reply:
I would probably say that if a word (or, in this case, compound) is in MW, use them as gospel